libs/dh/metering-point/feature-overview/src/lib/secondary-master-data/dh-secondary-master-data.component.html

```diff
@@ -160,10 +136,8 @@
           {{ t('capacity') }}
         </dt>
         <dd class="watt-text-m">
-          <ng-container *ngIf="secondaryMasterData?.capacity; else emptyField">
-            {{ secondaryMasterData?.capacity }}
-            {{ t('units.kilowatt') }}
-          </ng-container>
+          {{ secondaryMasterData?.capacity | emDashFallback }}
```

Comment:
Question: wouldn't this change cause kilowatt to be shown even though capacity is not defined?

Reply:
Yes. good catch. One solution could be to make the `capacity` a param to the `units.kilowatt` translation. Another solution is to make a getter and return the display value from it or return `undefined` if `capacity` is `null`/`undefined`. A third solution is to also create a directive that displays the em dash.

Do you have a preference? I think I'll try the getter first.

Reply:
Went with the first option.